misc/install.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+ENIGMA_NODE_VERSION=4.4
```

Comment:
4.4 was intentional based on the current installation instructions - will
wait for Bryan to say confirm testing Enigma with 4.5 before changing
anything...

On 25 Aug 2016 4:25 p.m., "Michael J. Ryan" <notifications@github.com>
wrote:

> In misc/install.sh
> <https://github.com/NuSkooler/enigma-bbs/pull/87#discussion_r76265276>:
>
> > @@ -0,0 +1,108 @@
> > +#!/usr/bin/env bash
> > +
> > +ENIGMA_NODE_VERSION=4.4
>
> If you just use 4 you'll now get 4.5, where 4.4 will only give you 4.4.x
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NuSkooler/enigma-bbs/pull/87/files/6c1a364864694fcf4b922e13f09f8ed61543570a#r76265276>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAozMP_ZiGO1HtduXb0iXvCHk7CXTs6aks5qjbPVgaJpZM4Jqsxw>
> .
>